Form-Controls/styles.css

```diff
@@ -0,0 +1,41 @@
+:root{
+    --blue:#54ADB6;
+    --black:#000000;
+    --brown:#641E16;
+    --yellow:#CC99FF;
+
+}
+
+body{color: var(--brown);
+font-family:'Raleway',sans-serif;
+font-size: 20px;}
+
+h1{text-align: center;
+color: var(--brown);}
+
+main{display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+padding: 20px;
+background-color: var(--blue);
+}
+
+#name{margin-bottom: 10px;}
```

Comment:
Typically, we avoid using #id rules in CSS. This is because of "specificity". A single id will always override an infinite number of classes, or elements, so it can make rules "stick" and be hard to override. 

https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4

Reply:
Thank you for this valuable information 🙏